src/Mod/Draft/importDWG.py

```diff
@@ -199,26 +198,29 @@ def get_oda_converter():
     if "ODAFileConverter" in path: # path set manually
         if os.path.exists(path) and os.path.isfile(path):
             return path
-    elif platform.system() == "Linux":
-        path = "/usr/bin/ODAFileConverter"
-        if os.path.exists(path) and os.path.isfile(path):
-            return path
     elif platform.system() == "Windows":
         odadir = os.path.expandvars("%ProgramFiles%\\ODA").replace("\\", "/")
         if os.path.exists(odadir):
             for sub in os.listdir(odadir):
                 path = odadir + "/" + sub + "/" + "ODAFileConverter.exe"
                 if os.path.exists(path) and os.path.isfile(path):
                     return path
+    elif platform.system() == "Linux":
+        path = "/usr/bin/ODAFileConverter"
+        if os.path.exists(path) and os.path.isfile(path):
+            return path
+    else: # for macOS
```

Comment:
There should be a check if macOS, else unknown system

Reply:
Ok. Let’s say we disagree about programming practices.

Reply:
In any case I don't know what we would do right now on a system that is not mac, linux or windows... I think if someone comes up with a new config, then we see about what paths to use...

Reply:
@Roy-043 FreeBSD fixes are submitted to FreeCAD from time to time. So while it is unofficial, it still supported  
https://github.com/search?q=repo%3AFreeCAD%2FFreeCAD%20freebsd&type=code